packages/js/experimental/src/inbox-note/action.tsx

```diff
@@ -56,26 +56,12 @@ export const InboxNoteActionButton: React.FC< InboxNoteActionProps > = ( {
 		onClick();
 	};
 
-	if ( variant === 'link' ) {
-		return (
-			<Button
-				className="woocommerce-inbox-note__action-button"
-				variant={ 'link' }
-				isBusy={ inAction }
-				href={ href ?? '#' }
-				onClick={ handleActionClick }
-			>
-				<span>{ label }</span>
-			</Button>
-		);
-	}
-
 	return (
 		<Button
 			className="woocommerce-inbox-note__action-button"
 			variant={ variant }
 			isBusy={ inAction }
-			disabled={ inAction }
+			{ ...( href ? { href } : {} ) }
```

Comment:
Maybe we shouldn't remove the `disabled` prop, because otherwise it can still trigger subsequent clicks and consequently new requests to `/wc-analytics/admin/notes/{note_id}`.

Nit: Also, maybe simplifying the `href` prop handling to `href={ href || undefined }` to make slightly more readable?

Reply:
> Maybe we shouldn't remove the disabled prop.

Good catch! It was not my intention to remove it! 

>  Also, maybe simplifying the href prop handling to href={ href || undefined } to make slightly more readable?

It makes sense!

Addressed both feedback with 25802722de745b84792b63e247a94156ec895f88! Thanks for the review!